src/js/ia/IADevPipeline.js

```diff
@@ -16,6 +16,9 @@
                 var iadp;
                 iadp = Handlebars.templates.dev_pipeline(data.dev_milestones);
                 $("#dev_pipeline").html(iadp);
+
+                // 100% width
+                $(".site-main > .content-wrap").first().removeClass("content-wrap").addClass("wrap-pipeline");
```

Comment:
looks dangerous.

At least, in the future someone will be wondering why the pipeline behaves so differently while attempting to implement the redesign.

Is there a way we can note this fact so when the time comes we won't forget?